packages-exp/auth-exp/src/platform_browser/persistence/local_storage.ts

```diff
@@ -209,7 +206,6 @@ class BrowserLocalPersistence
   }
 
   _addListener(key: string, listener: StorageEventListener): void {
-    this.localCache[key] = this.storage.getItem(key);
```

Comment:
i think you still need this, it warms up the cache from existing data when we register a listener. without this you could have a situation like:
- add value
- attach listener
- delete value
and the listener wouldn't fire

Reply:
nvm, you'll have written to the cache regardless during the initial write

Reply:
Cache is always updated on any data changes (add/put/remove) regardless of # listeners so there is no need for this step. It only complicates things.

Reply:
Actually, some proactive populating helps reduce spuriously triggering and fixes some of the tests. I've added a conditioned clause back and to indexedDb to stay consistent.